packages/backend/server/src/plugins/copilot/tools/edit-doc.ts

```diff
@@ -0,0 +1,79 @@
+import { tool } from 'ai';
```

Comment:
should file name `doc-edit.ts`? keep rule as the exists file `web-search.ts`.

Reply:
lgtm